### PR TITLE
Add python-tool and remove duplicate

### PR DESCRIPTION
**Pull request by JaekwanLee**
Add python-tool and remove duplicates

@mchow @agentiqd This repo needs python-tool as a submodule and I added necessary files into repo. Will Jenkins automatically pick it up and fetch the submodule?

### CODE REVIEW

File: dags/agent_event_metrics.py [hunk 1]
```diff
@@ -6,9 +6,10 @@
 from datetime import datetime, timedelta
 
 from logger import logger
-from utils.file_utils import dump_to_csv_file, load_csv_file
-from utils.aws_util import s3_upload_file, s3_download_file
-from utils.aws_util import create_etl_daily_key_prefix
+from utils.aws_helper import create_etl_daily_key_prefix
+from tools.utils.aws_util import s3_upload_file, s3_download_file
```
Comment (akspha) [inline dags/agent_event_metrics.py:10]: I see this file is removed, where is the coming from now, python tools ?

Comment (JaekwanLee) [reply dags/agent_event_metrics.py:10]: Yes

File: docker-compose-LocalExecutor.yml [hunk 2]
```diff
@@ -19,11 +18,11 @@ services:
         environment:
             - LOAD_EX=n
             - EXECUTOR=Sequential
-            - AIQ_AIRFLOW_DB_HOST=${AIQ_AIRFLOW_DB_HOST}
-            - AIQ_AIRFLOW_DB_PORT=${AIQ_AIRFLOW_DB_PORT}
-            - AIQ_AIRFLOW_DB_USERNAME=${AIQ_AIRFLOW_DB_USER}
-            - AIQ_AIRFLOW_DB_NAME=${AIQ_AIRFLOW_DB_NAME}
-            - AIQ_AIRFLOW_DB_PASSWORD=${AIQ_AIRFLOW_DB_PASSWORD}
+            - AIQ_AIRFLOW_DB_HOST=postgres
```
Comment (slaveniq) [inline docker-compose-LocalExecutor.yml:21]: why are we doing away with env variables?

Comment (JaekwanLee) [reply docker-compose-LocalExecutor.yml:21]: this docker compose is to run test locally and independently. Previously, it worked okay as our ~/.agentiq/demo.env does not contain those values but it now gets collided by local environment variables.  So I removed them .

**Pull request merged**